internal/dump/acl_test.go

```diff
@@ -21,13 +21,27 @@ func Test_acl_decode(t *testing.T) {
 			},
 			want: "user::rw-\nuser:0:rwx\nuser:65534:rwx\ngroup::rwx\nmask::rwx\nother::r--\n",
 		},
+		{
+			name: "decode group",
+			args: args{
+				xattr: []byte{2, 0, 0, 0, 8, 0, 1, 0, 254, 255, 0, 0},
+			},
+			want: "group:65534:--x\n",
+		},
 		{
 			name: "decode fail",
 			args: args{
 				xattr: []byte("abctest"),
 			},
 			want: "",
 		},
+		{
+			name: "decode fail",
```

Comment:
Name copy-pasted from above. "empty"?

Reply:
I've changed the name to `decode empty fail`.